packages/pg-v5/commands/credentials/url.js

```diff
@@ -12,7 +12,9 @@ async function run(context, heroku) {
 
   let db = await fetcher.addon(app, args.database)
   let cred = flags.name || 'default'
-  if (util.essentialPlan(db) && cred !== 'default') {
+  if (util.essentialNumPlan(db)) {
+    throw new Error('You can’t perform this operation on Essential-tier databases.')
+  } else if (util.legacyEssentialPlan(db) && cred !== 'default') {
```

Comment:
can we combine these?
```suggestion
  if ((util.essentialNumPlan(db) || util.legacyEssentialPlan(db)) && cred !== 'default') {
```

Reply:
yes combined those ^